site/content/entry/yuzu-progress-report-jan-2021/index.md

```diff
@@ -0,0 +1,283 @@
+These three objects are thus used to represent a single "signallable event".
+
+These mechanisms would be used, for example, when a game needs to play a sound.
+The audio service will produce a `KEvent` that will "signal" when it's ready for more audio data from a game, and this will communicate with the audio thread so that data can be submitted and requested as needed by using these `KReadableEvent` and `KWritableEvent` attributes.
+
+As always, the major benefit of these rewrites is accuracy.
+These changes will also make it easier in the long term to add the code of future updates that Nintendo might do to their kernel, since the behaviour of this implementation will be closer to the hardware and the existing code won’t need many changes to accommodate for it.
+
+There’s still room for more additions and improvements, so stay tuned for more work on this area.
+
+After a period of inactivity, [FernandoS27](https://github.com/FernandoS27) is back with a PR to [optimize clock calculations](https://github.com/yuzu-emu/yuzu/pull/5275) by introducing a new method to compute time values, such as nanoseconds, microseconds, milliseconds, and also guest clock and cpu cycles, from the native clock of the CPU.
+This is a relatively small change, but since these calculations are performed often, all optimizations are helpful to reduce the time spent on these transformations.
+
+## Input changes
+
+What can be better than QoL changes that you didn’t know you wanted until you saw them?
+[german77](https://github.com/german77) gives us just that by [animating the controller input settings image](https://github.com/yuzu-emu/yuzu/pull/5339) on player actions.
+You can see the results yourself:
+
+{{< single-title-imgs
+    "I like to move it, move it!"
+    "./pro.mp4"
+    "./joy.mp4"
+  >}}
+
+The animation will highlight the buttons that are currently being pressed.
+There’s also a second animation for the analog sticks to make the calibration of deadzones and range more intuitive.
+Your controller will behave in-game as shown by the animation in the settings, so if you think there’s any problem with your input, it’s a good idea to check if it’s working as intended here.
+
+Input lag, the worst enemy of competitive games… And [german77](https://github.com/german77) comes to the rescue, by [preventing overscheduling events.](https://github.com/yuzu-emu/yuzu/pull/5861)
+
+`Pokémon Let’s Go Eevee/Pikachu` may disconnect the emulated controller, or fail to recognise the connection.
+By [applying a delay to this process](https://github.com/yuzu-emu/yuzu/pull/5805), [german77](https://github.com/german77) improves the stability of input emulation for this series of games.
+Handheld or single Joy-Con input modes are still needed to play this game, be sure to check your input settings in `Emulation > Configure… > Controls. Swap the default Pro Controller to, for example, Handheld`.
```

Comment:
Remove the backtick from `Handheld` and move it to after `Emulation > Configure… > Control`